### PR TITLE
feat: add doc link for labels (resolve #413)

### LINKED ISSUE #413: PR 报告的 Labels 处提供指向文档的超链接
建议在 PR 报告的 Labels 处提供指向文档的超链接，方便检查分类是否符合要求

### PR DESCRIPTION
**Pull request by AnzhiZhang**
resolve #413

### CODE REVIEW

**Comment by alex3236** [comment]
为啥不直接把 Labels 改蓝（

File: .github/workflows/scripts/utilities.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def report_plugin(plugin: Plugin, tag: Tag) -> str:
         _check('fetch repository', failures)
     )
     report += _row(
-        'Labels',
+        'Labels ([Doc](https://docs.mcdreforged.com/en/latest/plugin_dev/plugin_catalogue.html#label) [文档](https://docs.mcdreforged.com/zh-cn/latest/plugin_dev/plugin_catalogue.html#label))',
```
Comment (AnzhiZhang) [inline .github/workflows/scripts/utilities.py:200]:
```suggestion
        '[Labels](https://docs.mcdreforged.com/en/latest/plugin_dev/plugin_catalogue.html#label)',
```

**Comment by alex3236** [comment]
草 合错了（

**Comment by AnzhiZhang** [comment]
怎么不reopen重新合啊

**Pull request closed without merging**